README.md

```diff
@@ -28,11 +18,19 @@ the source code given only:
   - or run ./prepare_dependencies.py cmake
```

Comment:
Line 18 and 19 are shown on a single line and I didn't realize I should run `./prepare_dependencies.py cmake` or `./prepare_dependencies.py` then `cmake`.